azure-pipelines.yml

```diff
@@ -47,6 +51,21 @@ jobs:
           ProjectDirectory: 'packages/common'
           Arguments: 'build'
 
+      - script: yarn deploy
+        displayName: 'Deploy Common Package'
+        condition: and(succeeded(), eq(variables.isPR, false))
+        env:
+          HOME: '$(Agent.BuildDirectory)'
+          BRANCH: '$(Build.SourceBranchName)'
+          PULL_REQUEST: '$(isPR)'
+          COMMIT_MESSAGE: '$(Build.SourceVersionMessage)'
+          PACKAGE_LOCATION: 'packages/common'
+          PACKAGE_NAME: 'common'
+          SITE_NAME: 'script-lab-react-storybook'
+          DEPLOYMENT_PASSWORD_SCRIPT_LAB_REACT_STORYBOOK_ALPHA: '$(DEPLOYMENT_PASSWORD_SCRIPT_LAB_REACT_STORYBOOK_ALPHA)'
+          DEPLOYMENT_PASSWORD_SCRIPT_LAB_REACT_STORYBOOK_BETA: '$(DEPLOYMENT_PASSWORD_SCRIPT_LAB_REACT_STORYBOOK_BETA)'
+          DEPLOYMENT_PASSWORD_SCRIPT_LAB_REACT_STORYBOOK_STAGING: '$(DEPLOYMENT_PASSWORD_SCRIPT_LAB_REACT_STORYBOOK_STAGING)'
+
   - job: EditorPipeline
```

Comment:
Seems like a lot of duplicate code in these jobs.  Couldn't we use Templates and parameters to factor this out?  https://docs.microsoft.com/en-us/azure/devops/pipelines/yaml-schema?view=azure-devops&tabs=schema%2Cparameter-schema#template-references

Reply:
I didn't know that was possible. Thanks for the resource Raul, that's super helpful to make the pipeline a lot cleaner